publiq-ui/pub-h1.stories.mdx

```diff
@@ -31,20 +21,8 @@ export const commonArgTypes = {
 
 <ArgsTable story="Default" />
 
-## Title
-
-<Canvas>
-  <Story
-    name="Title"
-    args={{ variant: Heading1Variant.TITLE }}
-    argTypes={commonArgTypes}
-  >
-    {Template.bind({})}
-  </Story>
-</Canvas>
-
 ```javascript
-<pub-h1 variant="Heading1Variant.TITLE">Heading 1</pub-h1>
+<pub-h1>Heading 1</pub-h1>
 ```
 
-<ArgsTable story="Title" />
+<ArgsTable story="Default" />
```

Comment:
We already show the `ArgsTable` on line 22 https://github.com/cultuurnet/udb3-frontend/pull/47/commits/8ae6bded0d324224fc12fb7140059b09025903d1#diff-d5175fa21352de12428f109d27859cfeR22

Reply:
But maybe that first one should be removed instead? As it's now above the example codeblock.